Makefile

```diff
@@ -17,15 +17,19 @@ precompile:
 	cp pkg/META.in pkg/META
 	ocamlbuild -use-ocamlfind -package topkg pkg/build.native
 
-build_without_utop: compile_error setup_convenient_bin_links precompile
+preprocess: precompile
```

Comment:
I don't get why the CI is failing right now. It says it can't find `Reason_parser_explain_raw` but this rule clearly should have been executed. Maybe if we include `compile_error: update_error precompile`, it'd work